frontend/pages/SoftwarePage/components/cards/SoftwareDetailsSummary/SoftwareDetailsSummary.tsx

```diff
@@ -32,6 +40,9 @@ interface ISoftwareDetailsSummaryProps {
   iconUrl?: string;
   /** Displays OS icon instead of Software icon */
   isOperatingSystem?: boolean;
+  /** Displays an edit CTA to edit the software's icon
+   * Should only be defined for team view of an installable software */
+  onClickEditIcon?: () => void;
```

Comment:
This is the same issue I mentions with `onDeleteFile` handler. seems fine unless it seems in the future we may want to show the icon irrespective if a handler is provided

Reply:
Same, I don't expect is to want to show a disabled icon for this